main.py

```diff
@@ -495,7 +502,7 @@ def _del_stored_flow_by_id(self, dpid, cookie, flow_id):
             if stored_flow["_id"] == flow_id:
                 index_deleted = i
                 break
-        if not index_deleted:
+        if index_deleted is None:
```

Comment:
I've ended up catching a bug here, if the index was to be `0` it would execute this early return.